pkg/ctl/commander.go

```diff
@@ -64,17 +67,17 @@ type Builder interface {
 // |___________|           |___________|           |___________|
 //
 func NewCommander(ctx context.Context, builder Builder, ctls ...Command) Commander {
-
-	c := &commander{
+	cmdr := &commander{
```

Comment:
If you change it here `c -> cmdr` assure to also change it in the member function definitions `func (c *commander)  -> func (cmdr *commander) `

Reply:
After taking a second look I think `c` is better. Will change back to that.